commands/create-account.js

```diff
@@ -1,11 +1,17 @@
-
 const exitOnError = require('../utils/exit-on-error');
 const connect = require('../utils/connect');
 const { KeyPair } = require('near-api-js');
+const NEAR_ENV_SUFFIXES = [
+    'near',
```

Comment:
I think this should go to config. Maybe even just reuse `masterAccount` for this, but use `helperUrl` for account creation when available.

Reply:
`helperUrl`? I thought that is only used in the dev-deploy. As far as I know, the `create_account` command will always need a `masterAccount` right?

Reply:
it is generally used when you create accounts using `near.createAccount` https://github.com/near/near-api-js/blob/f1dcf73ff3549fe7bc7caf35c7c6e43fbc4a2703/src/near.ts#L59
https://github.com/near/near-api-js/blob/f1dcf73ff3549fe7bc7caf35c7c6e43fbc4a2703/src/near.ts#L37

Though really the only hard statement here is that name suffix should be configurable, I'm not sure it should be universally be the same as `masterAccount`.

Reply:
Gotcha, interesting. Currently if you try passing just the  `helperUrl` yargs is set to fail because `masterAccount` is required.

`./bin/near create_account mike.asdf.test --accountHelper https://helper.testnet.nearprotocol.com`
gives
>Missing required argument: masterAccount
I feel like that'd be a good enhancement for another issue. Is it cool if we do this in another PR?

Reply:
I do like the idea of using config for those, by the way, and I'll be pushing up a change that checks config's `networkId` against it.

Reply:
>  and I'll be pushing up a change that checks config's networkId against it.

not sure what do you mean

Reply:
> I feel like that'd be a good enhancement for another issue. Is it cool if we do this in another PR?

sure, my main point here is just so that we avoid hardcoding suffixes in shell. If they are in config – they can easily be altered for a given project, etc. Probably should also be passable as argument (again so that it's easy to override when needed).

Reply:
I agree, I just pushed up a commit that essentially _suggests_ to the user that most of the time if they're on betanet, the account will end in .beta. But it's not blocking. We can't make it a hard requirement because of the custom TLAs, too. Good point